packages/jest-snapshot/src/colors.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// https://jonasjacek.github.io/colors/
+// https://webaim.org/resources/contrastchecker/
+
+// Contrast 6.89 AA
+export const aForeground2 = 90; // #870087
+export const aBackground2 = 225; // #ffd7ff
```

Comment:
```suggestion
export const aBackground2 = 0xd7 | 0xff | 0xd7;
```

again, just a suggestion which would mean we wouldn't need the comment

Reply:
These are indexes into arbitrary color table, so I got deleted the comments.